docs/README.md

```diff
@@ -59,3 +59,21 @@ Notes (Optional):
 - You need to have LaTeX installed for rendering math equations. You can for example install TeXLive by doing one of the following:
   - on Ubuntu (Linux) run `apt-get install texlive` or otherwise follow the instructions on the TeXLive website
   - use the [RTD docker image](https://hub.docker.com/r/readthedocs/build)
+
+## Developing docs
+
+When developing the docs, building docs can be VERY slow locally because of the notebook tutorials.
+To speed this up, enable this flag in `path/to/pytorch-lighting/docs/conf.py`
```

Comment:
@kaushikb11 let's do this with the env variable so we do not leak some personal changes to the codebase

Reply:
@Borda but when developing you have to run this command all the time:
```bash
grunt --project=docs
```

i don't know how grunt works under the hood, but i assume env vars will still be available to the python scripts it eventually calls?

Reply:
Yup, commented below with suggestions! Thanks